#
# On POWER this patch also fixes test-ildoubl and test-ldouble failures where tan
# rounded toward zero had acceptable 1 ULP error. Upstream is using 3 ULP, but
# we prefer to keep the bound tighter unless we have a reason not to.
#
# This is the generic power version of this patch that applies cleanly when the
# ppc64le patches are not being applied. See glibc-ppc64le-46.patch for the ppc64le
# version.
#
--- glibc-2.17-c758a686/sysdeps/powerpc/fpu/libm-test-ulps	2014-07-26 20:28:21.120919701 -0400
+++ glibc-2.17-c758a686/sysdeps/powerpc/fpu/libm-test-ulps	2014-07-26 20:47:17.657901758 -0400
@@ -2641,6 +2641,9 @@
 ifloat: 1
 ildouble: 2
 ldouble: 2
+Test "tan_towardzero (2) == -2.1850398632615189916433061023136825434320":
+ildouble: 1
+ldouble: 1
 Test "tan_towardzero (3) == -0.1425465430742778052956354105339134932261":
 float: 1
 ifloat: 1
